programs/bpf/c/sdk/bpf.mk

```diff
@@ -102,8 +107,10 @@ help:
 	@echo ''
 	@echo 'User settings'
 	@echo '  - The following setting are overridable on the command line, default values shown:'
-	@echo '    - Show commands while building:'
-	@echo '      V=1'
+	@echo '    - Show commands while building: V=1'
+	@echo '      V=$(V)'
+	@echo '    - Use LLVM from docker: DOCKER=1'
```

Comment:
probably should document somewhere around here that the user needs to `docker pull solanalabs/llvm`

Reply:
`ci/docker-run.sh` does this, but there's also cases when you don't want to pull (you're on an airplane with no uplink) so if we autopull then we'd also want to add a way to disable it.  My vote for now is to just document that the user must pull